internal/provisioner/kops_provisioner.go

```diff
@@ -151,6 +157,18 @@ func (provisioner *KopsProvisioner) CreateCluster(cluster *model.Cluster) error
 		return err
 	}
 
+	// Set the ELB tags for the public subnets
+	if provisioner.publicSubnetIds != "" {
```

Comment:
nit: we normally do the `strings.Split` immediately after getting the flag value and then pass the `[]string` around, but this is totally fine!